en/docs/build-and-run/build-and-run.md

```diff
@@ -1,69 +1,78 @@
-# Build and run the program
+# Build and run a Ballerina program
 
-Follow the steps below to create a sample Ballerina program in VS Code. 
+You can build your Ballerina program and run it either by using the **Code** view or the **Diagram** view when you use the Ballerina extension.
 
-## Use the Code View
+## Use the Code view
 
->**Note:** Skip 1-3 steps if you already have a Ballerina code.
+Let's build and run a sample Ballerina program using the Code view.
+
+>**Note:** If you already have a Ballerina code that you would like to build and run, you can skip steps 1 to 3 given below.
   
-1. Click **View** in the menu bar of the editor, and click **Command Palette**.
+1. Clik **View** > **Command Palette** on the VS Code editor.
 
-    >**Tip:** You can use the shortcut methods `⌘ + ↑ + P` on Mac and `Ctrl + Shift + P` on Windows and Linux.
+    >**Tip:** You can use the shortcut methods `⌘ + ↑ + P` on MacOS and `Ctrl + Shift + P` on Windows and Linux.
 
-2. In the search bar, type `Show Examples`, and click **Ballerina: Show Examples**.
+2. In the search bar, type `Show Examples` to get the list of available Ballerina samples.
 
-3. Select the **Hello World Main** example.
+3.  Click **Ballerina: Show Examples** and select the **Hello World Main** example.
 
-4. Run the program.
+4. Run the program using one of the following options:
 
     - **Option 1**
       
-        Click on the **Run** CodeLens on the editor. 
+        Click the **Run** CodeLens on the editor:
```

Comment:
Can we combine the two lines into one? for better clarity and to reduce unnecessary space usage?

E.g.

- **Option 1** - Click the **Run** CodeLens on the editor: